src/pages/Cart/Cart.js

```diff
@@ -1,59 +1,25 @@
-import React, { useState } from 'react';
-import { AiOutlineHeart, AiFillQuestionCircle } from 'react-icons/ai';
-import { BsTrash } from 'react-icons/bs';
-import CartItem from './CartItem';
+import React, { useState, useEffect } from 'react';
+import CartItemLst from './CartItemList';
+import CartSummary from './CartSummary';
 import './Cart.scss';
 
 const Cart = () => {
-  const [cartData, setCartData] = useState([]);
-  const [isHovering, setIsHovering] = useState(false);
-
-  const openTooltip = () => {
-    setIsHovering(true);
-  };
-
-  const closeTooltip = () => {
-    setIsHovering(false);
-  };
+  const [cartItemList, setCartItemList] = useState([]);
+  useEffect(() => {
+    fetch('data/CartMockData.json')
+      .then(result => result.json())
+      .then(data => setCartItemList(data));
```

Comment:
```suggestion
      .then(setCartItemList);
```

들어오는 인자와 콜백함수에서 받는 인자 값이 동일하면 위와 같이 축약형으로도 사용할 수 있습니다!

Reply:
맞습니다 ! 저번에 .then(console.log)에서 설명 한 번 해주셨습니다.
축약해서 진행할까요 ?!